cmd/crane/cmd/root.go

```diff
@@ -115,6 +123,7 @@ func New(use, short string, options []crane.Option) *cobra.Command {
 
 	root.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "Enable debug logs")
 	root.PersistentFlags().BoolVar(&insecure, "insecure", false, "Allow image references to be fetched without TLS")
+	root.PersistentFlags().DurationVar(&dialTimeout, "dial-timeout", dialTimeout, "Modify the dial timeout used to contact the registry.")
```

Comment:
Sorry for the runaround, but I think we can probably just remove this flag until we find out we need it. If you'd prefer I'm happy to make this change myself after we merge this, before we cut another release. Let me know.

(Total aside, I'm starting to hate these global flags. `crane login --insecure --platform=windows/arm64` does nothing, and sounds unnecessarily scary.)

Reply:
I'm happy to kill it.  Gimme a few minutes to tweak things

Reply:
Ok RFAL

Reply:
> (Total aside, I'm starting to hate these global flags. crane login --insecure --platform=windows/arm64 does nothing, and sounds unnecessarily scary.)

Ack, we should find a way to make these not-quite-global.